### ISSUE TITLE
Download Packs (tv shows) (improvement) 

### ISSUE THREAD

**Issue by Tinuz97** [Tikipeter/repository.tikipeter#378]
In the past you had a 'test' download pack, premiumize only.
That did work great for me (only manually unzip, but that is no issue on a computer). 

After some time you did remove it, because if I remember correctly, you where busy with other options, bug fixing etc, and did remove it, to rewrite this option (if I am correct)

Any plans on bringing back this feature (the first test did work great for me with premiumize). 

Or a option maybey in the 'browse pack' that is available already, that you can download every episode separately (holding OK button on let's say Episode 1, 2 etc) to download each episode (in the pack option) 

Just  a idea/improvement 😄

**Comment by Tinuz97** [comment]
With 'hold the OK button' I mean this option, to download every episode from that pack separately. (or a download complete pack option). 
![Screenshot_20210204-004511](https://user-images.githubusercontent.com/47715606/106824133-6fa81480-6682-11eb-97ba-1e43ce4d2e4a.jpg)

**Comment by Tikipeter** [comment]
I appreciate the reminder Tinuz, I did kind of forget about this.

The easy thing with Premiumize was that it offered a zip download. RD and AD don't offer this in their API, at least RD don't from what I can remember. So I was a little stuck in offering a standard way to download a PACK.

I think in the end I will do something similar to your picture 2; let the user select the files from the pack they wish to download. The issue with that is if the PACK is a large amount of files, then the user selects all those files, they will all be downloaded at the same time. I don't have a mechanism in place to limit the amount of simultaneous downloads.

I need to finalize what I'm going to do and then get it done, as it's been a while now with no progress.

**Comment by Tikipeter** [comment]
This should finally be done for the next update.

**Comment by Tinuz97** [comment]
@Tikipeter 

Choose Files to Download :   Kies Bestanden om te Downloaden
Multi File Pack Download Started... :    Downloaden Multi Bestandspakket Gestart ...

**Comment by Tinuz97** [comment]
Are there more translations needed that we forgot @Tikipeter?

I noticed Browse Packs as a example. 
That one should be:  Blader door pakketten

**Comment by Tikipeter** [comment]
That looks like an error on my part.
I did have a translation for "Browse Debrid Packs".
I took the "Debrid" part out.
Let me see if I stuffed up the translation files somehow..

**Comment by Tinuz97** [comment]
Now that I look at it, after the update, I think : Download pakket is better.
No need to hurry, just for a next update.
So Blader door pakketten, change to Blader door pakket.

**Comment by Tinuz97** [comment]
Forgot to say, but I LOVE this option!
It does make a nice folder with the show name, a folder with season(s), the way I like it. 

Works very good, thx! 
![HiddenIllegalCormorant-small](https://user-images.githubusercontent.com/47715606/108134246-28744780-70b6-11eb-91c3-14f7830b858b.gif)

**Comment by Tikipeter** [comment]
Hard to do that correct season code, but hopefully it works OK checking the file's name and getting it from there if the pack is a full show pack. Time will tell.

The main issue I have at the moment is I had to disable notifications for these pack downloads. If you choose any decent number of files and I left notifications working for packs, then the notification would be showing constantly, and it would fall behind and continue showing long after the downloads were finished. I haven't decided what to do about that in the future, so I just disabled notifications for any pack downloads.
